docker-compose.fast.yml

```
@@ -20,7 +20,12 @@ services:
       start_period: 30s
       start_interval: 5s
 
-  yaptide_worker:
+  yaptide_simulation_worker:
+    healthcheck:
+      start_period: 60s
+      start_interval: 5s
+
+  yaptide_helper_worker:
     healthcheck:
       start_period: 60s
       start_interval: 5s
```

```suggestion
  yaptide_helper_worker:
    healthcheck:
      start_period: 30s
      start_interval: 3s
```

I think it can be shorter as this one doesn't require to download any simulators...